benchmarks/fiber_swfft.cpp

```diff
@@ -0,0 +1,51 @@
+
+#include "fiber_swfft.h"
```

Comment:
We should keep original library name: SWFFT and swfft.h

Reply:
There is no "original library".  See above.

Reply:
What I mean is that we can keep the name SWFFT when building the library using https://git.cels.anl.gov/hacc/SWFFT, 
fiber_swfft would sound like a different implementation. Also, the name fiber is a placeholder for now.
Using swfft is also more compact and inline with internal enumeration access.

Reply:
SWFFT is the original library. FIBER_SWFFT is the integration piece missing in the original code. We only own FIBER name space. All the SWFFT names live in HACC name space which is the name of their application. There is no SWFFT name anywhere in the code. Only their `README` file contains the name. There is no reason to pretend there is a header file called `swfft.h`. That's why we create our own `fiber_swfft.h` that we own and can modify at any time.

Reply:
OK, it makes sense.